automation/jleechanorg_pr_automation/jleechanorg_pr_monitor.py

```
@@ -326,6 +331,37 @@ def _should_skip_pr(self, repo_name: str, branch_name: str, pr_number: int, curr
         self.logger.info(f"⏭️ Skipping PR {repo_name}/{branch_name}#{pr_number} - already processed commit {current_commit[:8]}")
         return True
 
+    def _resolve_automation_username(self) -> str:
+        """Resolve the automation username from multiple sources."""
+        # 1. Explicitly passed (CLI)
+        if self._explicit_automation_username:
+            self.logger.debug(f"👤 Using explicit automation username: {self._explicit_automation_username}")
```

Ruff (0.14.11)

Logging statement uses f-string

(G004)